project/AppDependencies.scala

```diff
@@ -33,8 +33,8 @@ object AppDependencies {
   def test(scope: String = "test, it"): Seq[sbt.ModuleID] = Seq(
     "org.scalatest"          %% "scalatest"          % "3.2.2"             % scope,
```

Comment:
scalatest should be bumped to 3.3.0-SNAP2. Also jettyFromWiremockVersion should be bumped to 9.4.35.v20201120. The one we are using currently is 2018 version.

Reply:
It is better to stick to main releases, as they are generally safer. I'll keep Scalatest at 3.2.2

Reply:
Ok, but it still needs to be bumped to 3.2.3 as that's the latest main release

Reply:
If that's the case then upgrading to 3.2.3 is acceptable, as we don't typically upgrade to non-main releases

Reply:
yep no worries

Reply:
https://github.com/hmrc/individuals-income-received-api/pull/208#discussion_r539203768